docs/prerequisites.md

```diff
@@ -1,43 +1,60 @@
-Requirements (PC)
+Requirements
 ==
+
+- An Android 8+ device (to install the patched APK on)
+
+### If patching on PC:
+
 - ADB
 - x86/x86_64 host architecture
 - Zulu JDK 17
 
+### If patching on Android:
+
+- arm64-v8a host architecture
+
 Common
 ==
-### 0. If you are preparing to install in **ROOT environment**, follow the method below
-- remove all YouTube related modules
-- remove directories: `/data/adb/service.d`, `/data/adb/post-fs-data.d`, `/data/adb/revanced`
-- reboot device
+
+> #### If you are preparing to install in a **ROOT environment**:
+>
+> 1. Remove all YouTube related modules.
+> 2. Remove the following directories: `/data/adb/service.d`, `/data/adb/post-fs-data.d`, `/data/adb/revanced`
+> 3. Reboot device.
 
 ​
-### 1. Check the [README.md](https://github.com/inotia00/revanced-patches/tree/revanced-extended#-json-format) of revanced-patches for supported versions
+### 1. Check the supported app versions
+
+ Check the JSON format in the [README.md](https://github.com/inotia00/revanced-patches/tree/revanced-extended#-json-format) of the revanced-patches repository to see which app versions are supported.
+
+YouTube:
 
 ![support_version_youtube](https://github.com/inotia00/revanced-documentation/blob/main/images/compatible_packages_youtube.png)
 
-→ YouTube
+
+YouTube Music:
 
 ![support_version_music](https://github.com/inotia00/revanced-documentation/blob/main/images/compatible_packages_youtube_music.png)
 
-→ YouTube Music
 
-![support_version_reddit](https://github.com/inotia00/revanced-documentation/blob/main/images/compatible_packages_reddit.png)
+Reddit:
 
-→ Reddit
+![support_version_reddit](https://github.com/inotia00/revanced-documentation/blob/main/images/compatible_packages_reddit.png)
```

Comment:
The screenshots in this section can easily be confused as pictures or dynamic renderings of the **currently** supported versions rather than examples that only show where the YouTube, YTM, and Reddit versions are specified in the JSON.

Reply:
It's better to add a note like:
```
Screenshots don't always show the currently supported version. Check this link for the latest supported version.
```